__tests__/app.test.js

```diff
@@ -86,4 +86,58 @@ describe("GET /api", () => {
             }
         });
     });
-})
\ No newline at end of file
+})
+
+describe("GET /api/articles", () => {
+    test("200: Returned items are in an array", () => {
+        return request(app)
+          .get("/api/articles")
+          .expect(200)
+          .then((res) => {
+            expect(Array.isArray(res.body.articles)).toBe(true);
+        });
+    });
+
+    test("200: Returned array has the right number of items", () => {
+        return request(app)
+          .get("/api/articles")
+          .expect(200)
+          .then((res) => {
+            console.log(res.body.articles);
+            expect(res.body.articles.length).toBe(13)
+        });
+    });
```

Comment:
This could all be included in the below test to save from having multiple tests.